scripts/demo.sh

```
@@ -357,56 +426,108 @@ echo -e "${GREEN}✓ DNS-based load balancing validated with pod scaling${NC}\n"
-      \"units\": 250,
+      \"units\": 100,
       \"nanos\": 0
     }
-  }
-}" localhost:50051 meridian.current_account.v1.CurrentAccountService/ExecuteDeposit)
+  },
+  \"payment_order_reference\": \"$PAYMENT_ORDER_REF\"
+}" localhost:50051 meridian.current_account.v1.CurrentAccountService/InitiateLien)
+
+LIEN_ID=$(echo "$LIEN1" | jq -r '.lien.lienId')
+LIEN_STATUS=$(echo "$LIEN1" | jq -r '.lien.status')
```

Shellcheck (0.11.0)

[warning] LIEN_STATUS appears unused. Verify use (or export if used externally).

(SC2034)